### PR TITLE
Added support for the LocationFiltersAcl

### PR DESCRIPTION
**Pull request by ronpal**
## Description
Please describe the change you have made.

## Checklist:
- [X] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [7.54.16] - 2024-08-22
+### Added
+- [Feature Preview]  Added LocationFiltersAcl capability.
```
Comment (doctrino) [inline CHANGELOG.md:23]: This is not a feature preview IMO. The group endpoint is already there and this is a supported ACL for that endpoint. This is a simple adding a missing ACL.

Feature preview are things which triggers a warning when you use them, or in the case of Toolkit, something you have to activate with a feature flag.

```suggestion
- Added new LocationFiltersAcl capability.
```

File: cognite/client/data_classes/capabilities.py [hunk 2]
```diff
@@ -797,6 +797,22 @@ class Scope:
...
+    scope: AllScope | IDScope
+    allow_unknown: bool = field(default=False, compare=False, repr=False)
+
+    class Action(Capability.Action):  # type: ignore [misc]
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        SpaceID = IDScope
```
Comment (doctrino) [inline cognite/client/data_classes/capabilities.py:813]: Have you tested this scope? IDScope is named `idScope` and looking at the spec it seems `id` is the correct name.
image

**Review by doctrino — COMMENTED** [review]
Just a few adjustments.

File: cognite/client/data_classes/capabilities.py (hunk 2 above)
Comment (ronpal) [reply cognite/client/data_classes/capabilities.py:813]: I think the "spec" is inaccurate, this is the json sent by Fusion: 

```{
            "locationFiltersAcl": {
              "actions": [
                "READ",
                "WRITE"
              ],
              "scope": {
                "idScope": {
                  "ids": [
                    "1"
                  ]
                }
              }
            }
          }```

**Review by doctrino — APPROVED** [review]

**Pull request merged**